### PR TITLE
Mental health champion config

### PR DESCRIPTION
**Pull request by DuttonMa**
Had to redo config split for mental health champion after recent changes to unity_base to put the 'base' config in project/sites/mentalhealthchampionni/config rather than project/sites/mentalhealthchampionni (this was necessary to make the config split work correctly)

### CODE REVIEW

File: project/config/mentalhealthchampionni/config/search_api.server.solr_default.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ backend_config:
   connector: standard
   connector_config:
     scheme: http
-    host: mentalhealthchampionni_solr
+    host: mentalhealthchampion_solr
```
Comment (omahm) [inline project/config/mentalhealthchampionni/config/search_api.server.solr_default.yml:43]: This should be the original mentalhealthchampionni_solr value

**Review by omahm — CHANGES_REQUESTED** [review]
Need to possibly run project:build again to get the right solr name

File: project/config/mentalhealthchampionni/config/search_api.server.solr_default.yml (hunk 1 above)
Comment (DuttonMa) [reply project/config/mentalhealthchampionni/config/search_api.server.solr_default.yml]: Wow, that's a good spot !

**Review by omahm — APPROVED** [review]
LG 👍

**Pull request merged**